### PR TITLE
Update test2_copy2 copy.js

### PR DESCRIPTION
**Pull request by xdan-arch**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
:white_check_mark: A review job has been created and sent to the PullRequest network.  
*****  
Check the status or cancel PullRequest code review [here](https://daniel-b0af.pullrequest.tech/dash/projects/1/xDanielA/test-repo/reviews).

**Review by ghost — COMMENTED** [review]
<table id="pr-rule-based-message">
    <thead>
        <tr>
            <th width="50"></th>
            <th width="100%">
                1 Message
            </th>
        </tr>
    </thead>
    <tbody>
                    <tr>
                <td>:books:</td>
                <td>
                    It looks like the description for this pull request is either blank or very short. Adding a high-level summary will help our reviewers provide better feedback. Feel free to include questions for PullRequest reviewers and make specific feedback requests.
                                    </td>
            </tr>
                    </tbody>
</table>
<p align="right">
    <a href="https://daniel-b0af.pullrequest.tech/dash/projects/1/xDanielA/test-repo">
        
    </a>
</p>

**Review by ghost — COMMENTED** [review]
aklajsdfkljasfl;ksadflsdasdf
[](#0b3e053a-b804-4158-b1a4-424ad8db6357 "PullRequest Meta [Do Not Modify]")
*****
Was this helpful? [Yes](https://daniel-b0af.pullrequest.tech/reviews/ratings/0b3e053a-b804-4158-b1a4-424ad8db6357/true) | [No](https://daniel-b0af.pullrequest.tech/reviews/ratings/0b3e053a-b804-4158-b1a4-424ad8db6357/false)

**Review by ghost — COMMENTED** [review]
testing comment on same lines after rename
[](#02bfe015-b0c5-42b9-b7db-60f5cf38f211 "PullRequest Meta [Do Not Modify]")
*****
Was this helpful? [Yes](https://daniel-b0af.pullrequest.tech/reviews/ratings/02bfe015-b0c5-42b9-b7db-60f5cf38f211/true) | [No](https://daniel-b0af.pullrequest.tech/reviews/ratings/02bfe015-b0c5-42b9-b7db-60f5cf38f211/false)

File: test2_copy2 copy.js [hunk 1]
```diff
@@ -14,3 +14,46 @@ console.log("B");
...
 console.log("C");
 debugger; console.log("D");debugger;if (foo == null) {bar();}
+
+
+debugger;
+
+console.log("E");
+
+function footwo() {
+  var bar;
```
[the remark this answers is not in the document]
Comment (xdan-arch) [reply test2_copy2 copy.js]: adding a comment from github

File: test2_copy2 copy_change.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+var x = 12;
+
+if (x===-0){
+  console.log("Hello");
+}
+
+while(true){
+}
```
Comment (xdan-arch) [inline test2_copy2 copy_change.js:8]: adding a new comment from github

File: test2_copy2 copy_change.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+console.log("C");
+debugger; console.log("D");debugger;if (foo == null) {bar();}
+
+
+debugger;
+
+console.log("E");
+
+function footwo() {
+  var bar;
```
[the remark this answers is not in the document]
Comment (xdan-arch) [reply test2_copy2 copy_change.js:24]: what if I leave a comment here.

File: test2_copy2 copy.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (xdan-arch) [reply test2_copy2 copy.js]: trying again

**Pull request merged**